docs/reference/modules/cluster/remote-clusters-security.asciidoc

```diff
@@ -0,0 +1,41 @@
+[[remote-clusters-security]]
+=== Configure remote clusters with security
+To use {ccr} or {ccs} with secured clusters, all clusters must have the {es} 
+security features enabled.
+
+The local cluster must be allowed to connect to the remote clusters, which means 
+that the certificate authority (CA) used to sign the keystore of the local
+cluster must be trusted by the remote clusters.
+
+User authentication is performed on the local cluster and the user and user’s 
+roles are passed to the remote clusters. A remote cluster checks the user’s
+roles against its local role definitions to determine which indices the user is 
+allowed to access.
+
+Before using {ccr} or {ccs} with secure {es} clusters, complete the following 
+configuration tasks:
+
+. Enable the {es} {security-features} on every node in each connected cluster by
+setting `xpack.security.enabled` to `true`. Refer to the
+<<general-security-settings,{es} security settings>>.
+
+. Encrypt internode communications with Transport Layer Security (TLS) on every
+node to establish a trust relationship between the local cluster and all remote
+clusters. Refer to 
+<<security-basic-setup,set up basic security for the {stack}>> for the required
+steps to configure security.
++
+NOTE: This procedure uses the same CA to generate certificates for all nodes.
+Alternatively, you can add the CA certificates from the local cluster as a
+trusted CA in each remote cluster. 
+
+. Connect <<remote-clusters-connect,remote clusters>> to a local cluster.
+
+. Configure <<remote-clusters-privileges,remote cluster privileges>>.
+
+[[remote-clusters-security-ccr]]
+==== {ccr-cap} and security
+After enabling and configuring security, connecting a remote cluster, and
+configuring privileges, you need to
+<<ccr-getting-started-follower-index,configure a follower index>> on your local
+cluster to replicate on a remote cluster.
```

Comment:
I only called out CCR because I didn't think that any additional steps were required for CCS. It does feel odd. I just wasn't sure what to add here. We could add some information like, "You can now [search across](https://www.elastic.co/guide/en/elasticsearch/reference/master/modules-cross-cluster-search.html) single or multiple remote clusters.", though that feels a little weak.